server/account/src/serviceOperations.ts

```diff
@@ -694,6 +695,10 @@ export async function listIntegrations (
 
   if (isAllowedService) {
     socialIds = socialId != null ? [socialId] : undefined
+    if (socialIds === undefined && account !== systemAccountUuid) {
```

Comment:
Let's better remove admin check from verifyAllowedServices and fix all places where admin: true is potentially used.